Makefile

```diff
@@ -98,7 +103,7 @@ ci-test: ## Run unit tests in CI
 .PHONY: backend-docker-start
 backend-docker-start:	## Starts a Docker-based backend for development
 	@echo "$(GREEN)==> Start Docker-based Plone Backend$(RESET)"
-	docker run -it --rm --name=backend -p 8080:8080 -e SITE=Plone $(DOCKER_IMAGE)
+	docker run -d -it --rm --name=backend -p 8080:8080 -e SITE=Plone $(DOCKER_IMAGE)
```

Comment:
We usually don't run the backend in daemon mode from this command, so this might be surprising. If you keep it, make sure you also add a command to stop it, and explain this in the readme.